<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>econ_ui</class>
 <widget class="QDialog" name="econ_ui">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>714</width>
    <height>498</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Engineering Economics</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="0">
    <widget class="QFrame" name="frame">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QGroupBox" name="input_box">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="title">
         <string>Input</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_3">
         <item row="0" column="1">
          <widget class="QComboBox" name="find_combo">
           <item>
            <property name="text">
             <string>Present Worth</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Future Worth</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Annual Worth</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Gradient</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Capital Cost</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="find_label">
           <property name="text">
            <string>What do you want to find?</string>
           </property>
           <property name="buddy">
            <cstring>find_combo</cstring>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="period_label">
           <property name="text">
            <string>Compounding periods:</string>
           </property>
           <property name="buddy">
            <cstring>period_edit</cstring>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QComboBox" name="know_combo">
           <item>
            <property name="text">
             <string>Present Worth</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Future Worth</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Annual Worth</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Gradient</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="3" column="2">
          <widget class="QLabel" name="period_info">
           <property name="text">
            <string>e.g. 6, 10</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QLineEdit" name="period_edit">
           <property name="text">
            <string>Enter a number</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="know_label">
           <property name="text">
            <string>What do you know?</string>
           </property>
           <property name="buddy">
            <cstring>know_combo</cstring>
           </property>
          </widget>
         </item>
         <item row="4" column="0" colspan="3">
          <widget class="QGroupBox" name="interest_box">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="title">
            <string>Interest Rate</string>
           </property>
           <layout class="QGridLayout" name="gridLayout_4">
            <item row="0" column="1">
             <widget class="QGroupBox" name="eff_rate_box">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="title">
               <string>Enter effective rate:</string>
              </property>
              <layout class="QGridLayout" name="gridLayout_6">
               <item row="0" column="0">
                <widget class="QLabel" name="rate_label">
                 <property name="text">
                  <string>Effective Interest Rate:</string>
                 </property>
                 <property name="buddy">
                  <cstring>rate_edit</cstring>
                 </property>
                </widget>
               </item>
               <item row="0" column="1">
                <widget class="QLineEdit" name="rate_edit">
                 <property name="text">
                  <string/>
                 </property>
                </widget>
               </item>
               <item row="2" column="0">
                <widget class="QLabel" name="info_label">
                 <property name="text">
                  <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-style:italic;&quot;&gt;Enter as a decimal (e.g. 0.04)&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item row="1" column="0" colspan="2">
             <widget class="QGroupBox" name="eff_calc_box">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="title">
               <string>Effective Interest Rate Calculation</string>
              </property>
              <layout class="QFormLayout" name="formLayout_2">
               <item row="0" column="1">
                <widget class="QLineEdit" name="nominal_rate_edit"/>
               </item>
               <item row="0" column="0">
                <widget class="QLabel" name="nominal_rate_label">
                 <property name="text">
                  <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Nominal Rate &lt;span style=&quot; font-style:italic;&quot;&gt;(as a decimal e.g. 0.04)&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
                 </property>
                 <property name="buddy">
                  <cstring>nominal_rate_edit</cstring>
                 </property>
                </widget>
               </item>
               <item row="2" column="0">
                <widget class="QLabel" name="eff_rate_label">
                 <property name="text">
                  <string>Effective Interest Rate: </string>
                 </property>
                 <property name="buddy">
                  <cstring>eff_rate_edit</cstring>
                 </property>
                </widget>
               </item>
               <item row="1" column="1">
                <widget class="QLineEdit" name="eff_period_edit"/>
               </item>
               <item row="2" column="1">
                <widget class="QLineEdit" name="eff_rate_edit">
                 <property name="enabled">
                  <bool>true</bool>
                 </property>
                 <property name="readOnly">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
               <item row="1" column="0">
                <widget class="QLabel" name="eff_period_label">
                 <property name="text">
                  <string>Compounding periods within the nominal rate:</string>
                 </property>
                 <property name="buddy">
                  <cstring>eff_period_edit</cstring>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item row="0" column="0">
             <widget class="QGroupBox" name="itype_box">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="title">
               <string>What type do you have?</string>
              </property>
              <layout class="QFormLayout" name="formLayout">
               <item row="0" column="0">
                <widget class="QRadioButton" name="effective_radio">
                 <property name="text">
                  <string>Effective</string>
                 </property>
                </widget>
               </item>
               <item row="1" column="0">
                <widget class="QRadioButton" name="nominal_radio">
                 <property name="text">
                  <string>Nominal</string>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QLineEdit" name="amount_edit">
           <property name="text">
            <string>Enter a number</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="amount_label">
           <property name="text">
            <string>Enter the present worth:</string>
           </property>
           <property name="buddy">
            <cstring>amount_edit</cstring>
           </property>
          </widget>
         </item>
         <item row="2" column="2">
          <widget class="QLabel" name="amount_unit">
           <property name="text">
            <string>($)</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QGroupBox" name="output_box">
        <property name="title">
         <string>Output</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_5">
         <item row="0" column="0">
          <widget class="QLabel" name="answer_label">
           <property name="text">
            <string>Answer equals:</string>
           </property>
           <property name="buddy">
            <cstring>answer_edit</cstring>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="answer_edit">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="1" column="0" colspan="2">
          <widget class="QTextEdit" name="output_info_box">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>find_combo</tabstop>
  <tabstop>know_combo</tabstop>
  <tabstop>amount_edit</tabstop>
  <tabstop>period_edit</tabstop>
  <tabstop>effective_radio</tabstop>
  <tabstop>nominal_radio</tabstop>
  <tabstop>rate_edit</tabstop>
  <tabstop>nominal_rate_edit</tabstop>
  <tabstop>eff_period_edit</tabstop>
  <tabstop>eff_rate_edit</tabstop>
  <tabstop>answer_edit</tabstop>
  <tabstop>output_info_box</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>econ_ui</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>econ_ui</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
